code/js/rcalc-math.js

```diff
@@ -0,0 +1,56 @@
+var isDivisionPerformed = false;
+
+// displays numerous inputs (numbers) on the screen
+function displayNumber(inputValue) {
+  nameCalculator.nameDisplayMain.value = nameCalculator.nameDisplayMain.value + inputValue;
+}
+
+// testing 0 input, when 0 comes after / buttons are disabled and message is given
+function testZero(inputValue) {
+  if (isDivisionPerformed && inputValue == 0) {
+    var buttonDisable = document.getElementsByClassName("rcalc-button");
+    for (var i = 0; i < buttonDisable.length; i++) {
+      buttonDisable[i].disabled = true;
+      buttonDisable[i].style.color = "black";
```

Comment:
For strings use constant as global property. ... var COLOR_BLACK .... then use this constant on all places ... do it for all colors and strings ""  ... STRING_EMPTY  ... it has memory reasons this changes

Reply:
const are used for colors and all strings and messages also messages such as const COLOR_BLACK, const STRING_EMPTY etc...